components/tools/OmeroPy/src/omero/cli.py

```diff
@@ -406,6 +406,23 @@ def handle_failed_request(self, rfe):
                     Default: $OMERO_USERDIR/tmp
 """
 
+SUDO_HELP = """
+The --sudo option is available to all
+commands accepting connection arguments.
+
+Below are a few examples showing how to use the sudo option
+
+Examples:
```

Comment:
Replace with ``Examples (admin or group owner only)``
this is in line with another command in ``bin/omero help <subcommand>`` suite.